src/services/userService.js

```diff
@@ -23,11 +29,56 @@ module.exports = {
       realLifeName,
       reputationScore,
       createdAt,
-      votingPower
+      votingPower,
+      steemBalance,
+      sbdBalance,
+      userVests,
     }
   },
 
-  getUserInfo: async function(name) {
+  getUserProfileInfo: async function(name) {
+    return this.getUserAccount(name)
+      .then(userInOurDb => {
+        return this.getDynamicVestInfo(userInOurDb);
+      })
+      .then(userInOurDb => {
+        return this.getCoinMarketCapPrices(userInOurDb);
+      })
+      .catch(err => {
+        new Error(err);
```

Comment:
We're initializing an Error object with another Error object here, and we aren't doing anything with it. Let's throw it:

```
.catch(err => {
   throw err;
}
```

Reply:
done